src/Components/Facility/FacilityCreate.tsx

```diff
@@ -505,6 +505,7 @@ export const FacilityCreate = (props: FacilityProps) => {
         crumbsReplacements={{
           [facilityId || "????"]: { name: state.form.name },
         }}
+        backUrl="/facility"
```

Comment:
Suppose I'm updating a facility (eg.: https://deploy-preview-3637--care-net.netlify.app/facility/c92cc4df-0fea-4d37-9c24-1c9f07bfac23/update) and press back, ideally I should land to https://deploy-preview-3637--care-net.netlify.app/facility/c92cc4df-0fea-4d37-9c24-1c9f07bfac23 instead of facilities page.

Reply:
[`85cc6d3` (#3637)](https://github.com/coronasafe/care_fe/pull/3637/commits/85cc6d36e26ab8b3fa99bcffbb73c9a94b708df8)